scripts/pylint_custom_plugin/tests/test_files/__init__.py

```diff
@@ -0,0 +1,7 @@
+from something import Something
+from something2 import something2 as somethingTwo
+
+__all__ = (
+    Something,
+    somethingTwo, #pylint: disable=aliasing-generated-code
+)
```

Comment:
Nit: missing newline

Reply:
Not saying we should do this now, but adding more test files for different test cases could be cool. Like to check that the error message includes the proper line, etc.

Reply:
I will make an issue for it